chaoscenter/authentication/api/handlers/rest/user_handlers_test.go

```diff
@@ -486,7 +490,7 @@ func TestResetPassword(t *testing.T) {
 			inputBody: &entities.UserPassword{
 				Username:    "testUser",
 				OldPassword: "",
```

Comment:
Great work!!, It would be great to add 1-1 testcase for invalid password too incase of `createUser, reserPassword, updatePassword...`

Reply:
Sure

Reply:
hey @Jonsy13,
We prefer not to add a negative test case for an invalid password in the `createUser` function. As per the new changes, we are removing the check for strict password criteria in `createUser` function. The user will be required to reset their password upon their initial login. Therefore, it would be unnecessary for the admin to enforce a strict password policy at the time of user creation, as the password will be reset anyway.